src/PlatformItemGroups.props

```diff
@@ -1,13 +1,6 @@
 <Project ToolsVersion="15.0">
 
   <PropertyGroup>
-    <_IsAndroid>false</_IsAndroid>
-    <_IsAndroid Condition="
-				$(IsMonoAndroid)
-				or '$(TargetFramework)'=='net6.0-android'
-				or '$(TargetFramework)'=='net7.0-android'
-				">true</_IsAndroid>
```

Comment:
This is the point of the changes. `IsMonoAndroid` is already defined in `Directory.Build.props`, and it is true for `net[6|7].0-android`.

I'm removing `_IsAndroid` property in favor of `IsMonoAndroid`, and renaming `IsMonoAndroid` to `IsAndroid`